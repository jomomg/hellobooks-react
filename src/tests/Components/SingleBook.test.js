import React from "react";
import SingleBookPage from "../../components/single_book";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

describe("<SingleBookPage/>", ()=>{
    const id = {
        params: {
            id: 1
        }
    };
    test("it renders successfully", ()=>{
        const single = shallow(<SingleBookPage match={id}/>);
        expect(shallowToJson(single)).toMatchSnapshot();
    });
});