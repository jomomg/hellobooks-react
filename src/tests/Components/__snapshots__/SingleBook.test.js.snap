// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingleBookPage/> it renders successfully 1`] = `
<div>
  <TopNav
    match={
      Object {
        "params": Object {
          "id": 1,
        },
      }
    }
    title="Book Info"
  />
  <div
    style={
      Object {
        "marginLeft": "24%",
        "marginRight": "1%",
        "marginTop": "1%",
      }
    }
  >
    <WithStyles(Grid)
      container={true}
      spacing={16}
    >
      <BookInfo
        book={Object {}}
        handleClick={[Function]}
        isAvailable={[Function]}
      />
      <SideInfo
        book={Object {}}
        isAvailable={[Function]}
      />
      <Notifier />
    </WithStyles(Grid)>
  </div>
</div>
`;
